src/main/java/org/gridsuite/cgmes/assembling/job/ProfilesAcquisitionJob.java

```diff
@@ -48,14 +48,19 @@ public static void handle(Boolean dependenciesStrictMode) {
                                                                          moduleConfigAcquisitionServer.getStringProperty("username"),
                                                                          moduleConfigAcquisitionServer.getStringProperty("password"));
              CgmesAssemblingLogger cgmesAssemblingLogger = new CgmesAssemblingLogger()) {
-
             acquisitionServer.open();
 
             cgmesAssemblingLogger.connectDb(moduleConfigCassandra.getStringProperty("contact-points"), moduleConfigCassandra.getIntProperty("port"));
 
             String casesDirectory = moduleConfigAcquisitionServer.getStringProperty("cases-directory");
             String acquisitionServerLabel = moduleConfigAcquisitionServer.getStringProperty("label");
 
+            // Get list of all tsos and business processes from cgmes boundary server
+            Set<String> tsos = cgmesBoundaryServiceRequester.getTsosList();
+            Set<String> businessProcesses = cgmesBoundaryServiceRequester.getBusinessProcessesList();
+            CgmesUtils.setNeededSourcingActors(tsos);
```

Comment:
maybe we can pass those sets as a parameters for functions that use them instead of creating a static sets in cgmesUtils

Reply:
Done